pubpub/pubpub
# Update: Terms, cookie banner

## Issue(s) Resolved
n/a. Updates privacy policy and privacy page to reflect removal of third-party tracking and ability for communities to add GA or simple. Updates cookie banner language to reflect same. Finally, 

## Test Plan
1. Visit `/legal/privacy` and make sure new GA language is present.
2. Visit `/legal/settings` and make sure new GA language is present
3. Clear cookies (or use incognito) and visit demo. You should see a privacy policy banner and then a cookie banner. Accept both, then refresh. You should see neither.
4. Clear cookies (or use incognito) and visit jtrialerror. You should only see the privacy policy banner.

## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas

### Supporting Docs
